build_defs/BUILD

```diff
@@ -0,0 +1,19 @@
+subinclude("//build_defs/sdk")
+
+protoc_plugins(
+    name = "proto_languages",
+    build_defs = CONFIG.PROTO.DEFINITIONS,
+    label_prefix = "proto_language:",
+)
+
+protoc_plugins(
+    name = "grpc_languages",
+    build_defs = CONFIG.PROTO.DEFINITIONS,
+    label_prefix = "grpc_language:",
+)
```

Comment:
These rules produce a subincludeable target which outputs a function based on the name parameter. This returns the proto language maps that we used to define in the .build_defs file directly.  

This is easily expandable to other types of protobuf targets e.g.:

```
protoc_plugins(
    name = "grpc_gateway_languages",
    build_defs = [...],
    label_prefix = "grpc_gateway_language:",
)
```